### PR TITLE
Add helper function that retrieves nextAvailableCommunitySlot

### PR DESCRIPTION
**Pull request by DeusExM4ch1n4**
This will help add clarity to the progress of the distribution of community allocated slots.

### CODE REVIEW

File: contracts/SL2RD.sol [hunk 1]
```diff
@@ -179,6 +179,11 @@ contract SL2RD is
         return _communitySplitsBasisPoints.value;
     }
 
+    /// @notice Returns the next available community slot id.
+    function nextAvailableCommunitySlot() public view returns (uint256) {
```
Comment (un7c0rn) [inline contracts/SL2RD.sol]: @DeusExM4ch1n4 Can we name this `countAllocatedCommunitySlots` or similar? I think count semantics will easier to reason about in the UI.

File: contracts/test/contract.SL2RD.test.js [hunk 2]
```diff
@@ -101,6 +101,11 @@ contract("SL2RD", (accounts) => {
         await splitContract.totalCommunitySlots(),
         (communitySplitsBasisPoints * ownerAddresses.length) / 10000
       );
+
+      assert.equal(
```
Comment (un7c0rn) [inline contracts/test/contract.SL2RD.test.js:106]: @DeusExM4ch1n4  Lets add a test for the non-zero case after an increment.

**Review by un7c0rn — APPROVED** [review]
LGTM

**Review by un7c0rn — CHANGES_REQUESTED** [review]
Sorry, stamped before seeing tests :)

File: contracts/test/contract.SL2RD.test.js [hunk 3]
```diff
@@ -109,7 +109,9 @@ contract("SL2RD", (accounts) => {
       );
 
       // Increment the current count of community slots.
-      await splitContract.transferNextAvailable(accounts[i]);
+      await splitContract.transferNextAvailable(
```
Comment (un7c0rn) [inline contracts/test/contract.SL2RD.test.js:112]: @DeusExM4ch1n4 Why a static address here rather than iterating over accounts?

Comment (DeusExM4ch1n4) [reply contracts/test/contract.SL2RD.test.js:112]: The original "i" was a typo. We just want to see if the counter actually works and keeps track of the increment as a helper function. We know from other tests that the logic of the transferNext itself is proper.

**Review by un7c0rn — APPROVED** [review]
LGTM
